### PR TITLE
opening foreign links in new tab

### PR DESCRIPTION
**Pull request by favourori**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -83,7 +83,7 @@ timeout = 300000
     parent = "docs"
     weight = 4
     identifier = "api-documentation"
-    url = "https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html" 
+    url = "https://www.javadoc.io/doc/org.apache.camel/camel-api/latest"
```
Comment (zregvart) [inline config.toml]: Not sure this needs to be changed, can you elaborate?

File: layouts/partials/header.html [hunk 2]
```diff
@@ -42,12 +42,16 @@
                             <a class="navbar-link" href="#">{{ .Name }}</a>
                             <div class="navbar-dropdown">
                                 {{ range .Children }}
+                                {{ if strings.HasPrefix .URL "https" }}
+                                <a class="navbar-item" href="{{ .URL | relURL }}" target="_blank" rel="noopener">{{ .Name }}</a>
```
Comment (zregvart) [inline layouts/partials/header.html]: This adds `target` and `rel` attributes for one link on the menu. What we need is something that will add this for all external links regardless of where they are rendered from.

[The issue](https://issues.apache.org/jira/browse/CAMEL-13888) in question has some pointers on how to accomplish that.

**Comment by favourori** [comment]
No problem. 

Let me work on that.

Best,
Favour

File: layouts/partials/header.html (hunk 2 above)
Comment (favourori) [reply layouts/partials/header.html]: Great. Will fix that today. Thanks for the pointers

**Comment by favourori** [comment]
Hi, check again.
I added blackfriday parser to config at the top markup

**Comment by favourori** [comment]
Hi @zregvart 

You can check now. All foreign links  open in new tabs now. Nav, footer, site-wide.

Thanks

File: antora-ui-camel/src/css/header.css [hunk 3]
```diff
@@ -429,4 +429,4 @@ body {
   #social-media {
     display: none;
   }
-}
```
Comment (zregvart) [inline antora-ui-camel/src/css/header.css]: Please revert this change

File: layouts/partials/header.html [hunk 4]
```diff
@@ -42,12 +42,16 @@
                             <a class="navbar-link navbar-topics" href="#">{{ .Name }}</a>
                             <div class="navbar-dropdown">
                                 {{ range .Children }}
+                                {{ if strings.HasPrefix .URL "https" }}
```
Comment (zregvart) [inline layouts/partials/header.html]: Please revert changes to this file

**Comment by favourori** [comment]
closing, so I can move on to the next issue. Should you need additional modifications, please let me know

**Comment by zregvart** [comment]
@favourori changes in pull requests need to be relating to a single topic, if your intent is to bundle multiple changes into a single pull request then we can't accept that. You need to work on separate topics in separate pull requests.

**Comment by favourori** [comment]
Okay. Noted
